src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1363,15 +1387,6 @@ internal virtual HttpResponseMessage GetResponse(HttpClient client, HttpRequestM
 
                         long requestContentLength = requestWithoutRange.Content is null ? 0 : requestWithoutRange.Content.Headers.ContentLength.Value;
 
-                        string reqVerboseMsg = string.Format(
-                            CultureInfo.CurrentCulture,
-                            WebCmdletStrings.WebMethodInvocationVerboseMsg,
-                            requestWithoutRange.Version,
-                            requestWithoutRange.Method,
-                            requestContentLength);
-
-                        WriteVerbose(reqVerboseMsg);
```

Comment:
Again Verbose. Please revert.

Reply:
I deduplicated this code by moving it to response. I will restore the function but in the deduplicated location.